### PR TITLE
Update GenX cases LP to MIP

### PR DESCRIPTION
**Pull request by joaquimg**

### CODE REVIEW

File: GenX/README.md [hunk 1]
```diff
@@ -16,3 +16,6 @@ GenX is licensed under the [GPL v2 license](https://github.com/GenXProject/GenX.
 
 The example cases in this directory are taken from
 [https://github.com/GenXProject/GenX.jl/tree/v0.4.1/example_systems](https://github.com/GenXProject/GenX.jl/tree/v0.4.1/example_systems).
+
+Cases were modified by setting `UCommit: 1` to consider unit commitment of operation
+variables, which is the source of integrality in GenX.
```
Comment (odow) [inline GenX/README.md]: You did't commit these?

Comment (joaquimg) [reply GenX/README.md]: not yet, it is failing to write the files, working on it

**Comment by joaquimg** [comment]
adding binaries this is failing due https://github.com/jump-dev/HiGHS.jl/issues/241

**Comment by odow** [comment]
Get rid of this:
https://github.com/jump-dev/open-energy-modeling-benchmarks/blob/841c545bac8de8eba03627c4c44877a5a5dc8353/GenX/cases/1_three_zones/settings/highs_settings.yml#L7

**Comment by joaquimg** [comment]
yup, but then it gets super slow to generate everything. so now I am at changing time limit as well

**Comment by joaquimg** [comment]
Problem 6 is a bit of a pain because they use DDP, so the 500s time limit is not enough for some subproblems

**Comment by odow** [comment]
Kill it. We don't need them all.

**Comment by joaquimg** [comment]
added just the first from 6 for now

**Pull request merged**